Resources/public/js/views/actions/ez-previewactionview.js

```diff
@@ -154,6 +154,21 @@ YUI.add('ez-previewactionview', function (Y) {
                     this.get('editPreview').set('version', val);
                     return val;
                 }
+            },
+
+            /**
+             * The languageCode
+             *
+             * @attribute languageCode
+             * @type {String}
+             * @default {}
+             */
+            languageCode: {
+                value: {},
```

Comment:
`@type {String}` but default value to `{}` ? default to `''` would be more suited